File: ios-app/KunlunStudentApp/Models/DormitoryModels.swift
import Foundation

struct DormitoryData: Decodable {
    let items: [DormitoryItem]
    let keyword: String
}

struct DormitoryItem: Decodable, Identifiable {
    let id: Int
    let dormitoryNo: String
    let weeklyScore: Double
    let monthlyScore: Double
}

File: android-app/app/src/main/java/com/kunlun/studentapp/data/model/DormitoryModels.kt
package com.kunlun.studentapp.data.model

data class DormitoryData(
    val items: List<DormitoryItem>,
    val keyword: String
)

data class DormitoryItem(
    val id: Int,
    val dormitory_no: String,
    val weekly_score: Double,
    val monthly_score: Double
)

